Source/Scene/ModelExperimental/FeaturePipelineStage.js

```diff
@@ -0,0 +1,133 @@
+import combine from "../../Core/combine.js";
+import defaultValue from "../../Core/defaultValue.js";
+import ShaderDestination from "../../Renderer/ShaderDestination.js";
+import FeatureStageVS from "../../Shaders/ModelExperimental/FeatureStageVS.js";
+
+/**
+ * The feature pipeline stage is responsible for handling features in the model.
+ *
+ * @namespace FeaturePipelineStage
+ * @private
+ */
+var FeaturePipelineStage = {};
```

Comment:
I know it won't all be implemented in this PR, but what's the intended scope of this stage. Is it only for feature picking, or is this the beginning of a broader metadata stage? Just trying to think if `FeaturePipelineStage` is the right name. E.g. is it really `FeaturePickingPipelineStage`?

Reply:
As discussed offline, it's deliberate decision to not to set the `pickId` here because this stage can be activated for styling (using the batch texture) even when `allowPicking` is `false`. This stage is for setting up the feature variables (`model_featureColor`, `model_featureSt`).